RecoEgamma/EgammaTools/interface/EGExtraInfoModifierFromValueMaps.h

```diff
@@ -104,10 +128,11 @@ EGExtraInfoModifierFromValueMaps(const edm::ParameterSet& conf) :
   ModifyObjectValueBase(conf) {
   constexpr char electronSrc[] =  "electronSrc";
   constexpr char photonSrc[] =  "photonSrc";
-
+  overrideExistingValues_ = conf.exists("overrideExistingValues") ? conf.getParameter<bool>("overrideExistingValues") : false;
```

Comment:
please insert this in all appropriate configurations instead of deriving a value from conf.exists .